tests/codeception/acceptance/Content/DefaultContentCest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * Class DefaultContentCest.
+ *
+ * @group content
+ */
+class DefaultContentCest {
+
+  /**
+   * Default content pages and meta data exist.
+   */
+  public function testExistingContent(AcceptanceTester $I) {
+    $pages = [
+      '/',
+      '/resources',
+      'research',
+      'about',
+      'page-not-found',
+      'access-denied',
+    ];
+
+    foreach ($pages as $page) {
+      $I->amOnPage($page);
+      $I->canSeeResponseCodeIs(200);
+    }
+
+    $I->amOnPage('/');
+    $I->canSee('Welcome to your site!');
+    $I->canSeeNumberOfElements('meta[property="og:image"][content*="/large/"]', 1);
+    $I->canSeeNumberOfElements('meta[property="og:image:url"][content*="/large/"]', 1);
+  }
+
+  /**
+   * There should be at least 15 media items.
+   */
+  public function testMedia(AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
```

Comment:
Perhaps we should change this to "Site Manager" due to the recent bug I introduced?

Reply:
That probably would make these fail. how about we leave it as is, and I'll adjust it or build more tests when I fix those permissions?

Reply:
Okie doke.